notebook2/notebook/notebook.py

```diff
@@ -223,50 +223,38 @@ def read_containers_status(container_statuses):
     if container_statuses is None:
```

Comment:
[The container_statuses must have length equal to the number of containers.](https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/V1PodStatus.md)

Reply:
(ergo it is never None)

Reply:
That's not really true:
<img width="350" alt="screen shot 2019-03-04 at 12 46 19 pm" src="https://user-images.githubusercontent.com/5543229/53754452-fbefa280-3e81-11e9-9d15-b33cce5bc5ec.png">

Reply:
Immediately after creation

Reply:
Presumably because no containers created yet